tests/Agent/IntegrationTests/IntegrationTests/AwsLambda/AwsLambdaAPIGatewayHttpApiV2ProxyRequestTest.cs

```diff
@@ -149,32 +149,33 @@ private void ValidateTraceHasParent(ServerlessPayload serverlessPayload)
         }
     }
 
-    public class AwsLambdaAPIGatewayHttpApiV2ProxyRequestTestNet6 : AwsLambdaAPIGatewayHttpApiV2ProxyRequestTest<LambdaAPIGatewayHttpApiV2ProxyRequestTriggerFixtureNet6>
+    public class AwsLambdaAPIGatewayHttpApiV2ProxyRequestTestNet8 : AwsLambdaAPIGatewayHttpApiV2ProxyRequestTest<LambdaAPIGatewayHttpApiV2ProxyRequestTriggerFixtureNet8>
```

Comment:
I wonder if we should just name all of these classes `NetOldest` and `NetLatest` so we don't have to come back and change this again in a couple years?

Reply:
I was planning to once the more pressing issues are hammered out, but it also would be a good Friday afternoon chore. 👍